packages/opc-styles/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "opc-styles",
```

The packages should be namespaced as per our npm org. So that we can publish them easily.

Also, @hybridx @gisumit Can you make sure all the components have that namespacing applied? Maybe we can add that in the generator itself.

```suggestion
  "name": "@one-platform/opc-styles",
```

---

@deshmukhmayur we already have it in the generator, here is an example https://github.com/1-Platform/op-components/pull/11/files#diff-66a29433b0951ef6f3f7e5b0b68b1dacR2

---

added the namespace to package.json file